BuddySave/Program.cs

```diff
@@ -49,21 +52,27 @@ private static async Task Main()
             Logger.Info("Exit");
         }
 
-        private static async Task Run(GameSave gameSave, Session session)
+        private static async Task Run(GameSave gameSave, Session session, ServerParameters serverParameters)
         {
             var input = string.Empty;
             while (!string.Equals(input, "exit", StringComparison.OrdinalIgnoreCase))
             {
-                Console.WriteLine("Waiting for action command (load, save, exit):");
+                Console.WriteLine("Waiting for action command (run, load, save, exit):");
                 input = Console.ReadLine();
+
                 switch (input?.ToLowerInvariant())
                 {
+                    case "run":
+                        await GamingSession.Run(gameSave, session, serverParameters);
+                        break;
+
                     case "load":
```

Comment:
Do we still need the `load` and `save` cases? `run` seems to cover both now.

Reply:
I am not sure if we need them for some conflict resolution, for example when lock stays after a crash. Additionally run requires more configuration than load/save flow.